src/services/wallet/index.js

```diff
@@ -0,0 +1,69 @@
+// @flow
+
+import { NativeModules } from 'react-native';
+import { BigNumber } from 'bignumber.js';
+import factory from '../../services/ethereum/factory';
+import type { WalletType } from '../../types/Wallet';
+
+export default class WalletService {
+  static async getWallets() {
+    const { Panthalassa } = NativeModules;
+    const walletAddress = await Panthalassa.PanthalassaEthAddress();
+    const walletArray = [{
+      currency: 'ETH', ethAddress: walletAddress, balance: null, name: 'Ethereum',
+    }, {
+      currency: 'PAT', ethAddress: walletAddress, balance: null, name: 'XPAT',
+    }];
+    return walletArray;
+  }
+
+  static async syncWallet(wallet: WalletType) {
+    // const container = await containerPromise;
```

Comment:
Does that function work?

Reply:
Not now, but will be used in a near future. That's why was kept like this